src/renderers/webgl/WebGLCubeMaps.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * @author mrdoob / http://mrdoob.com/
+ */
+
+import { CubeReflectionMapping, CubeRefractionMapping, CubeUVReflectionMapping, CubeUVRefractionMapping, EquirectangularReflectionMapping, EquirectangularRefractionMapping } from '../../constants.js';
+import { WebGLCubeRenderTarget } from "../WebGLCubeRenderTarget.js";
+
+function WebGLCubeMaps( renderer ) {
+
+	let cubemaps = new WeakMap();
+
+	function mapTextureMapping( texture, mapping ) {
+
+		if ( mapping === EquirectangularReflectionMapping ) {
+
+			texture.mapping = CubeReflectionMapping;
+
+		} else if ( mapping === EquirectangularRefractionMapping ) {
+
+			texture.mapping = CubeRefractionMapping;
+
+		}
+
+	}
+
+	function get( texture ) {
+
+		if ( texture && ( texture.isCubeTexture || texture.isWebGLCubeRenderTargetTexture ) ) {
+
+			return texture;
+
+		} else if ( texture && texture.isTexture ) {
+
+			const mapping = texture.mapping;
+			const isDeprecatedEquirectangular = mapping === EquirectangularReflectionMapping || mapping === EquirectangularRefractionMapping;
+
+			if ( mapping === CubeUVReflectionMapping || mapping === CubeUVRefractionMapping ) {
+
+				return texture;
+
+			} else if ( texture.isEquirectangularTexture || isDeprecatedEquirectangular ) {
```

Comment:
In the current PR, where are instance of `EquirectangularTexture` actually created^^? 

It seems right now, the renderer only looks for `EquirectangularReflectionMapping` and `EquirectangularRefractionMapping`. 

If these constant are considered to be deprecated, how would the user define an equirectangular texture then?

Reply:
> In the current PR, where are instance of `EquirectangularTexture` actually created^^? 

No example is using `EquirectangularTexture` yet. I was waiting to get some consensus in #19845 before investing more.

Reply:
Understood!

Reply:
Lets leave the `EquirectangularTexture` for another PR 😅